official/transformer/utils/dataset.py

```diff
@@ -67,6 +67,16 @@
 _BOUNDARY_SCALE = 1.1
 
 
+NUM_EXAMPLES = {
+    tf.estimator.ModeKeys.TRAIN: 4572160,
+    # TODO(robieta): Find out why the actual number is slightly lower.
```

Comment:
nit: Todos.

Also, @k-w-w , do you know?

Reply:
Yes, I believe the real number appears smaller because some examples are filtered out by the dataset (where the sequences have lengths > max_length)